docker/README.md

```diff
@@ -1,20 +0,0 @@
-Docker configuration for testing and development of Opaque.
```

Comment:
This diff shouldn't be here  since it's compared to the current master.

Reply:
I think I got this resolved.